src/web/data/userMapData.json

```diff
@@ -501,19 +581,23 @@
       "BITS Pilani",
       "Charotar University of Science & Technology",
       "Doon Business School",
+      "Galgotias University",
       "Gujarat University",
       "Guru Gobind Singh Indraprastha University",
       "Ideas That Scale Accelerator LLP",
       "IIIT",
+      "IIITD",
```

Comment:
Use full forms

Reply:
Would you like me to break down the abbreviation for IIIT or not? if not then Ill just keep it as IIIT Delhi, as thats what the D represents

Reply:
Don't modify the existing entries for now. Just write the unabbreviated version for IIITD.